File: src/main/java/com/sjs/lootbotga/game/cards/PirateValue.java
package com.sjs.lootbotga.game.cards;

public enum PirateValue implements CardValue{
	ONE (1,2),
	TWO (2,4),
	THREE (3,4),
	FOUR (4,2);

	private Integer value;
	private Integer amount;

	private PirateValue(Integer value, Integer amount) {
		this.value = value;
		this.amount = amount;
	}

	public Integer getValue() {
		return value;
	}

	public Integer getAmount() {
		return amount;
	}
}

File: src/main/kotlin/com/sjs/lootbotga/game/cards/PirateValue.kt
package com.sjs.lootbotga.game.cards

enum class PirateValue(private val value: Int, private val amount: Int) : CardValue {
    ONE(1, 2),
    TWO(2, 4),
    THREE(3, 4),
    FOUR(4, 2),
    CAPTAIN(999, 1);

    override fun getValue(): Int {
        return value
    }

    override fun getAmount(): Int {
        return amount
    }
}